examples/chart/teleport-cluster/.lint/initcontainers.yaml

```diff
@@ -0,0 +1,14 @@
+clusterName: helm-lint
```

Comment:
this file is identical to `resources.yaml`

Reply:
Good spot.